sql/sqlserver/server.go

```diff
@@ -94,10 +93,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Send the SQLRequest for SQL execution.
-	if httpStatus, err := s.execute(args, reply); err != nil {
-		http.Error(w, err.Error(), httpStatus)
-		return
-	}
+	s.Send(sqlwire.Call{Args: args, Reply: reply})
```

Comment:
Very strange for this method to be named `Send`. The `Send` name makes sense in the KV context because the request is being forwarded through a `TxnCoordSender`. Here we're going to execute the request directly. I think `executeCmd` would be more appropriate.